evm/db/__init__.py

```diff
@@ -17,6 +17,7 @@ def get_db_backend_class(import_path=None):
     return import_string(import_path)
 
 
-def get_db_backend(import_path=None):
+def get_db_backend(import_path=None, db_path=None):
```

Comment:
I think this should be `get_db_backend(import_path=None, **init_kwargs)` and then instantiate the backend with `backend_class(**init_kwargs)`.